src/applications/search/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appName": "Search",
+  "entryFile": "./search-entry.jsx",
+  "entryName": "search",
+  "rootUrl": "/search",
+  "production": false,
+  "contentPage": "search/index.md"
+}
```

Comment:
Because of some the work with the build script, the `contentPage` and `production` are actually deprecated now and can be removed.